modules/hcp-nodepool-hc.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-nodepool-hc_{context}"]
+= Creating a NodePool object for the hosted cluster
+
+A `NodePool` is a scalable set of worker nodes that is associated with a hosted cluster. `NodePool` machine architectures remain consistent within a specific pool and are independent of the machine architecture of the control plane.
+
+.Procedure
+
+. Create a YAML file with the following information about the `NodePool` object, replacing values as necessary:
++
+[source,yaml]
+----
+apiVersion: hypershift.openshift.io/v1beta1
+kind: NodePool
+metadata:
+  creationTimestamp: null
+  name: <hosted_cluster_name> <1>
+  namespace: <hosted_cluster_namespace> <2>
+spec:
+  arch: amd64
+  clusterName: <hosted_cluster_name>
+  management:
+    autoRepair: false <3>
+    upgradeType: InPlace <4>
+  nodeDrainTimeout: 0s
+  platform:
+    type: Agent
```

Comment:
This is also bare metal agent specific.

Reply:
Yes. This procedure is in the bare-metal-specific section of the disconnected docs. It is not part of the procedures for OpenShift Virtualization.